tests/test_mq/test_mq.py

```diff
@@ -305,12 +305,16 @@ def test_describe_broker_unknown():
 
     with pytest.raises(ClientError) as exc:
         client.describe_broker(BrokerId="unknown")
-    err = exc.value.response["Error"]
-    assert err["Code"] == "NotFoundException"
+    resp = exc.value.response
+    error = resp["Error"]
+    metadata = resp["ResponseMetadata"]
+    assert metadata["HTTPStatusCode"] == 404
```

Comment:
For exceptions that do map to a particular status code, adding an assert to ensure that the serializer is outputting the right one is recommended.